site/management.md

```diff
@@ -360,46 +384,51 @@ curl -i --header "authorization: Bearer &lt;token&gt;" http://localhost:15672/ap
 
-It is possible to configure which OAuth 2.0 scopes RabbitMQ should claim when redirecting the user to the authorization server.
-
-If `management.oauth_enabled = true` and `management.oauth_scopes` is not set, RabbitMQ default to `openid profile`.
+When using `management.oauth_enabled = true`, it is still possible to authenticate
+with [HTTP basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication)
+in the Management UI.
 
-Depending on the Authorization server, we may use regular expression in scopes, e.g. <*resource_server_id*>`.*`, or
-instead we have to explicitly ask for them, e.g.:
-  * <*resource_server_id*>`.tag:administrator`
-  * <*resource_server_id*>`.read:*/*/*`
+By the default, `management.oauth_disable_basic_auth` has the value `true`, meaning that when OAuth 2 is
+enabled, the Management UI only accepts OAuth 2 authentication. The Management UI shows a button with the label **Click here to login** like shown in the screenshot below:
+<img src="./img/oauth2/management-oauth.png" alt="Single OAuth 2.0 resource, with oauth_disable_basic_auth = true" style="width:75%;height:75%"/>
 
+To support OAuth 2.0 and Basic Authentication, set the
+ `management.oauth_disable_basic_auth` configuration key to `false`:
 
-### Configure OpenID Connect Discovery endpoint
+<pre class="lang-ini">
+...
+management.oauth_disable_basic_auth = false
+...
+</pre>
+The Management UI shows now a username/password login form for Basic Authentication in addition to the **Click here to login** button for OAuth 2 authentication:
 
-By default, RabbitMQ assumes the OpenID Connect Discovery endpoint is at `<management.oauth_provider_url>/.well-known/openid-configuration`. If your endpoint differs, you can set yours via the `management.oauth_metadata_url` setting.
+<img src="./img/oauth2/management-oauth-with-basic-auth.png" alt="Single OAuth 2.0 resource, with oauth_disable_basic_auth = false" style="width:75%;height:75%"/>
 
-RabbitMQ uses this endpoint to discover other endpoints such as **token** endpoint, **logout** endpoint, and others.
 
-### Logout workflow
+### <a id="#about-logout-workflow" class="anchor" href="#about-logout-workflow">About Logout workflow</a>
 
 RabbitMQ follows the [OpenID Connect RP-Initiated Logout 1.0 ](https://openid.net/specs/openid-connect-rpinitiated-1_0.html)
 specification to implement the logout workflow. This means that the logout workflow is triggered from the Management UI when the user clicks on the **Logout** button. Logging out from RabbitMQ management UI not only logs the user out from the management UI itself but also from the Identity Provider.
 
 There are other two additional scenarios which can trigger a logout. One scenario occurs when the OAuth Token expires. Although RabbitMQ renews the token in the background before it expires, if the token expires, the user is logged out.
```

Comment:
I will clarify. It meant that even if the management ui automatically renews the token, it may not do it on time due to various reasons like the browser freezes, the oauth server does not respond on time, etc. So, yes, the management ui will always try its best to renew the token. But ultimately, if the user performs an action and RabbitMQ sees the token has expired, the user is kicked out.